governance/parameter-docs/param-dai-savings-rate.md

```diff
@@ -0,0 +1,53 @@
+# Dai Savings Rate
+
+```
+Alias: DSR
+Parameter Name: Pot_dsr
+Containing Contract: Pot
+Scope: System
+Technical Docs: https://docs.makerdao.com/mcd-developer-guides/developer-guides-and-tutorials#dai-savings-rate-dsr
+```
+
+## Description
+The Dai Savings Rate allows Dai holders to deposit Dai into the contract and earn interest paid in Dai. Maker Governance can alter the rate at which interest accrues.
+
+The Dai Savings rate is usually expressed as a percentage and can be any value greater than 0%. Therefore, the DAI Savings rate cannot be negative.
+
+The Dai Savings Rate can also be turned off by Maker Governance - rendering the interest rate in effect to be 0%.
+
+## Purpose
+Maker Governance can utilise the Dai Savings Rate to encourage Dai holders to deposit Dai to the Dai Savings Rate contract. Higher amounts of Dai within the Dai Savings Rate
+should reduce the market supply of Dai and act as a means to control the Peg.
+
+## Trade-offs
+In essence, a higher Dai Savings Rate results in upward pressure on Dai price, and a lower Dai Savings Rate will reduce this pressure.
+
+The higher the Dai Savings Rate, the greater the incentive to deposit DAI and earn interest. However, removing Dai from the markets may lower liquidity in certain liquidity 
```

Comment:
Again, try to be really clear as to what the trade-offs are. Increased DAI demand versus higher cost to the protocol. Increased DAI demand versus lowering liquidity.

Try to say what is essential without too much extra. Focus on the trade-offs when increasing versus decreasing, rather than arbitrary high and low values. Basically, to increase the DSR is to trade-off revenue / maybe liquidity for DAI demand (which should increase the value of DAI.) To decrease it is to trade-off DAI demand for revenue, however, this might be a pure good if peg is high. 

Also maybe comment on growth as well. It's not just the effect on the peg. It makes DAI more attractive in general as an asset - which makes it easier to expand to new markets, convince users to use DAI instead of competitors, etc.